### ISSUE TITLE
Disabled focal length

### ISSUE THREAD

**Issue by KruemelGames** [NyokoDev/PhotoModePreserve#1]
This sadly disables the focal length option in the photo mode (camera tab).
What i tried: Started disabled this mod to figure out if its this mod. And focal lenght worked again. Disabled FirstPersonView (with chances some camera settings) too but sadly this wasnt the cause.
What happens: In the camera mode i enabled the focal length and changed the value but nothing happens
What should happen: the focal lenght should adjust while changing the value

Maybe some other options are bugged too, didnt test it yet.
Hope this can be fixed pretty easy :) Thanks for the mod

Edit: I tried all the other options in the camera tab and they all dont seem to work. Tried all the other tabs (lens, color, weather, environment) and they all work. 
Edit 2: This also disables the angle follow while following a car the angle doesnt change anymore like in the vanilla game.

**Comment by GIS-PuppetMaster** [comment]
enable the surround mode will fix this, you can disable it then, this mod still works normally

**Comment by GIS-PuppetMaster** [comment]
no, I'm wrong, the focal length seems only work in surround mode

**Comment by NyokoDev** [comment]
Use Color Adjustments instead

Reason: Preserve Photo Mode reeenables the rendering system causing the issues you mentioned as the 'PhotoModeRenderSystem' is very complex.

Now, the ColorAdjustments mod is a fresh rewrite modifying the same values from the ColorAdjustments Tab only. Lets you save and load these settings so you can keep them.

Efforts are underway for CA and main purpose is to improve that mod instead of Preserve Photo Mode

Preserve Photo Mode is not deprecated, however, CA is the better alternative

**Comment by GIS-PuppetMaster** [comment]
> Use Color Adjustments instead使用颜色调整代替
> 
> Reason: Preserve Photo Mode reeenables the rendering system causing the issues you mentioned as the 'PhotoModeRenderSystem' is very complex.原因：保留照片模式重新启用渲染系统，导致您提到的问题，因为“PhotoModeRenderSystem”非常复杂。
> 
> Now, the ColorAdjustments mod is a fresh rewrite modifying the same values from the ColorAdjustments Tab only. Lets you save and load these settings so you can keep them.现在，ColorAdjustments mod 是一个全新的重写，仅修改 ColorAdjustments 选项卡中的相同值。允许您保存和加载这些设置，以便您可以保留它们。
> 
> Efforts are underway for CA and main purpose is to improve that mod instead of Preserve Photo ModeCA 正在努力，主要目的是改进该模式而不是保留照片模式
> 
> Preserve Photo Mode is not deprecated, however, CA is the better alternative保留照片模式并未被弃用，但是 CA 是更好的选择

Yes I'm using this mod too, but I find out that this mod can be useful when you want to build  with a short focal length, so I'm using both of them

**Comment by GIS-PuppetMaster** [comment]
> Use Color Adjustments instead使用颜色调整代替
> 
> Reason: Preserve Photo Mode reeenables the rendering system causing the issues you mentioned as the 'PhotoModeRenderSystem' is very complex.原因：保留照片模式重新启用渲染系统，导致您提到的问题，因为“PhotoModeRenderSystem”非常复杂。
> 
> Now, the ColorAdjustments mod is a fresh rewrite modifying the same values from the ColorAdjustments Tab only. Lets you save and load these settings so you can keep them.现在，ColorAdjustments mod 是一个全新的重写，仅修改 ColorAdjustments 选项卡中的相同值。允许您保存和加载这些设置，以便您可以保留它们。
> 
> Efforts are underway for CA and main purpose is to improve that mod instead of Preserve Photo ModeCA 正在努力，主要目的是改进该模式而不是保留照片模式
> 
> Preserve Photo Mode is not deprecated, however, CA is the better alternative保留照片模式并未被弃用，但是 CA 是更好的选择

Also, this mod can frozen the time so I can build and simulate in morning, noon, and afternoon, the skylight of morning is much more comfortable compared to just disable day-night shifting, which lock the time at noon.
